js_modules/dagster-ui/packages/ui-core/src/ui/VirtualizedItemListForDialog.tsx

```diff
@@ -36,20 +36,15 @@ export function VirtualizedItemListForDialog<A>({
           // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
           const assetKey = items[index]!;
           return (
-            <Row
-              $height={size}
-              $start={start}
-              key={key}
-              ref={rowVirtualizer.measureElement}
-              data-key={key}
-            >
-              <Box
-                style={{height: '100%'}}
-                flex={{direction: 'row', alignItems: 'center'}}
-                border={itemBorders && index < items.length - 1 ? 'bottom' : null}
```

Comment:
The `height:100%` here was basically making the rowVirtualizer.measureElement just measure the estimatedSize